infra/base-images/base-builder/install_go.sh

```diff
@@ -26,3 +26,16 @@ echo 'Set "PATH=$PATH:/root/.go/bin:$GOPATH/bin"'
 
 go get -u github.com/mdempsky/go114-fuzz-build
 ln -s $GOPATH/bin/go114-fuzz-build $GOPATH/bin/go-fuzz
+
+go install golang.org/dl/gotip@latest \
```

Comment:
I'm confused. Youre installing gotip in base-builder but using it in base-runner. Is this intended?

Reply:
`gotip` is also being used base-builder in the coverage build in `compile_native_go_fuzzer`

Reply:
Right but where is it coming from in base-runner then?